gamemodes/terrortown/gamemode/client/cl_scoring.lua

```diff
@@ -25,8 +25,25 @@ CLSCORE.Players = {}
 CLSCORE.EventDisplay = {}
 CLSCORE.StartTime = 0
 CLSCORE.Panel = nil
-
-surface.CreateFont("WinHuge", {font = "Trebuchet24", size = 72, weight = 1000, shadow = true})
+CLSCORE.ScorePanelColor = Color(150, 50, 50)
+CLSCORE.ScorePanelNames = {
```

Comment:
I'm aware that all variables in this file seem to be names this way, but shouödn't they all start with a lowercase letter? Or do they have to stay this way for compatibility reasons?

Reply:
They are exposed, so people can change them. I just used the same names, as the PR does, for compatibility, but we could change them, if wanted, as we are already breaking many other things and this is not a big compatibility decrease.

Reply:
I say keep it that way. I'll rework the panel at some point, so we don't have to do this twice